examples/streetlights-operation-security.yml

```
@@ -78,9 +88,15 @@ channels:
       message:
         $ref: '#/components/messages/turnOnOff'
       security:
+      # This operation level security implies the ability to subscribe messages to 
```

WDYT about the following 2 rewordings:

```suggestion
      # This operation level security implies the ability to subscribe to messages from
```